Doc/data/refcounts.dat

```diff
@@ -2283,6 +2283,11 @@ PyType_CheckExact:PyObject*:o:0:
 PyType_FromSpec:PyObject*::+1:
 PyType_FromSpec:PyType_Spec*:spec::
 
+PyType_FromModuleAndSpec:PyObject*::+1:
+PyType_FromModuleAndSpec:PyObject*:module:0:
```

Comment:
Shouldn't this be `+1`? The function does `Py_XINCREF` on the module object and stores it in `ht_module`.

Reply:
Good Catch, Thanks, updated.